README.md

```diff
@@ -17,19 +17,16 @@ Install with [CocoaPods](http://cocoapods.org). Add to your `Podfile`:
 pod 'SSDynamicText', :head # YOLO
 ```
 
-## SSDynamicLabel
+## Example usage
 
-A label that responds when the user changes her preferred text size. Check out `Example` for a full example.
+You can create dynamic auto-sizing label, button, text field, and text view using a custom font and a base font size or a font descriptor. Check out full [example](https://github.com/splinesoft/SSDynamicText/blob/master/Example/SSDynamicTextExample/SSViewController.m).
 
-```objc
+UIKit views that responds when the user changes her preferred text size:
+
+### SSDynamicLabel
```

Comment:
Should we use inline code style here, as with `NSAttributedString` below?

### `SSDynamicLabel`